lib/npm.js

```diff
@@ -194,6 +194,13 @@ class Npm {
 
     await this.time('npm:load:configload', () => this.config.load())
 
+    const { Chalk, supportsColor, supportsColorStderr } = await import('chalk')
+    this.#noColorChalk = new Chalk({ level: 0 })
```

Comment:
Why are we attaching a new private attribute here that we turn around and only use to conditionally assign to other private attributes and never use otherwise/